aws/logs_monitoring/lambda_function.py

```diff
@@ -393,16 +433,58 @@ def generate_metadata(context):
     return metadata
 
 
+def extract_metric(event):
+    """Extract metric from an event if possible"""
+    try:
+        metric = json.loads(event['message'])
+        required_attrs = {'m', 'v', 'e', 't'}
```

Comment:
Where can I see the code related to that ?

Reply:
Nevermind found it: https://github.com/DataDog/datadog-lambda-layer-python/pull/5/files
I didn't see it because it isn't merged yet 👍